include/libretro.h

```diff
@@ -1248,6 +1248,16 @@ enum retro_mod
                                             * default when calling SET_VARIABLES/SET_CORE_OPTIONS.
                                             */
 
+#define RETRO_ENVIRONMENT_GET_PREFERRED_HW_RENDER 56
+                                           /* unsigned * --
```

Comment:
This should be the enum type, any reason this is an unsigned?

Reply:
Should also comment what HW_CONTEXT_NONE means here.

Reply:
Wanted to avoid to directly use an enum as arg cause it's quite a bad habit (some cores can be linked with -fno-short-enums causing strange behaviours interacting with RetroArch compiled without it for example).

Reply:
Funky, but ok if there is precedence.